### PR TITLE
Update meta info

### PR DESCRIPTION
**Pull request by do-you-dare**
This is part of https://github.com/mezuro/prezento/issues/350.

### CODE REVIEW

**Comment by rafamanzo** [comment]
This build shall pass after merging https://github.com/mezuro/likeno/pull/24

File: HACKING.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+### Ruby version
+
+  `2.3.0`
+
+  You can easily install it through the Ruby Version Manager - RVM. Instructions on how to do it can be found at http://rvm.io
+
+  It is expected to work with:
+
+  - 2.1.5 Debian 8
+  - 2.0.0 CentOS 7
```
Comment (rafamanzo) [inline HACKING.md]: Those are no longer supported now. But we still can say 2.2.2 and further are.

File: README.md [hunk 2]
```diff
@@ -72,4 +67,41 @@ It returns a `hash` with the response body.
...
 
-Kolekti follows the [semantic versioning](http://semver.org/) policy.
+Likeno follows the [semantic versioning](http://semver.org/) policy.
+
+### License
+
+Copyright (c) 2013-2016 The Author developers.
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU Affero General Public License as published by
```
Comment (rafamanzo) [inline README.md]: This uses the Lesser not Affero GPL :(

Comment (do-you-dare) [reply README.md]: Sorry for that; I copied it from another repository from mezuro, tough I can't remember which one now.

**Comment by rafamanzo** [comment]
Nice job!

**Pull request merged**